shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala

```diff
@@ -687,6 +699,10 @@ object GlutenConfig {
       (SPARK_S3_USE_INSTANCE_CREDENTIALS, "false"),
       (SPARK_S3_IAM, ""),
       (SPARK_S3_IAM_SESSION_NAME, ""),
+      (SPARK_S3_RETRY_MAX_ATTEMPTS, "3"),
+      (SPARK_S3_CONNECTION_MAXIMUM, "96"),
```

Comment:
`96`/`3` can also be the default value for the config and use something like `AWS_S3_CONNECT_TIMEOUT.defaultValueString` as well

Reply:
Yes. These keys can also use `defaultValueString` but existing code also use the value directly for easy reading unless it's a new config in gluten itself. So let's keep it.